jsx/App/locale/LocaleConstants.jsx

```diff
@@ -0,0 +1,210 @@
+// Put language constants here to use in your display.
+export const ENGLISH = "en";
+export const ESPANOL = "es";
+export const FRANCAIS = "fr";
```

Comment:
SG! Will you do the merging?

Reply:
Up to you guys. If you think you'll be working on the project for another few months, it would be good for you to do all the merges so you can learn how it works and be able to do future merges independently. If you don't want to mess with it, I can do it myself. 

The whole merge process takes ~10 minutes once you know what you're doing. Unless you get red X's, in which case you should ask for help and expect it to take longer. Here's the process: 

- merge this PR. Ideally, choose "squash and merge" to combine everything into a single commit, which will make the history of the master branch more readable. 
- wait ~60 seconds for the Build & Deploy task to finish, then make sure it finished with a green checkmark rather than a red X. 
- use the BrownCLPS site to check that things are working (you can skip this if it's a small change or if you're very confident)
- pick one of the language-specific repos to update first, let's say you pick Cofan-ALDP. 
- Open the GitHub page for the Cofan-ALDP repo. Go to the Actions tab, then click "Prepare Upgrade" in the left sidebar. Click the green "run workflow" button that appears on the right side of the page, choose the master branch, and run it. 
- Wait ~60 seconds for the Prepare Upgrade task to finish, then make sure it finished with a green checkmark rather than a red X.
- There should now be a new pull request called "Upgrade LingView core". Go to the Pull Requests tab and click on it. 
- Look at the Files Changed and make sure it looks ok: no unexpectedly changes, no merge conflicts. If something's wrong, make additional commits on that branch to fix it. 
- Merge the "Upgrade LingView core" PR. 
- Repeat all of that for the YucatecMaya repo. 
- Close the issue (if it wasn't auto-closed already) and if you want to, tell everybody that it's done! 

I ought to put these instructions in the Ultimate Guide and/or on the wiki, but for now I think the merge instructions only exist here.

Reply:
For this PR specifically, don't merge the "Upgrade LingView core" PR yet. We'll need to make additional commits so that all the AboutPage, GlossaryPage, etc. keep their content but get translated correctly, and so that things are in A'ingae rather than English. The default language should be Spanish. Many of the A'ingae translations will initially be missing, and that's ok.